consensus/spos/bls/subroundStartRound.go

```diff
@@ -136,6 +136,9 @@ func (sr *subroundStartRound) initCurrentRound() bool {
 
 	sr.AppStatusHandler().SetStringValue(common.MetricConsensusRoundState, "")
 
+	// get previous bitmap before generating next consensus group
+	previousBitmap := sr.GenerateBitmap(SrSignature)
```

Comment:
handle scenario on start of epoch block where the consensus group will still be selected based on previous model